lte/gateway/c/core/oai/tasks/nas/esm/esm_data_context.c

```diff
@@ -47,15 +47,6 @@ void free_esm_bearer_context(esm_ebr_context_t* esm_ebr_context) {
     if (NAS_TIMER_INACTIVE_ID != esm_ebr_context->timer.id) {
       esm_ebr_timer_data_t* esm_ebr_timer_data = NULL;
       nas_timer_stop(&(esm_ebr_context->timer));
-      /*
-       * Release the retransmisison timer parameters
-       */
-      if (esm_ebr_timer_data) {
-        if (esm_ebr_timer_data->msg) {
-          bdestroy_wrapper(&esm_ebr_timer_data->msg);
-        }
-        free_wrapper((void**) &esm_ebr_timer_data);
-      }
```

Comment:
why is this not needed anymore? Line 48 should be `esm_ebr_timer_data_t* esm_ebr_timer_data =esm_ebr_context->args` and we should keep the lines from 50 through 58, I think.

Reply:
Updated this